app/src/main/res/drawable/background_rounded_corners_ripple.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ripple android:color="#AAAAAAAA" xmlns:android="http://schemas.android.com/apk/res/android">
+<ripple android:color="?colorControlHighlight" xmlns:android="http://schemas.android.com/apk/res/android">
     <item android:drawable="@drawable/background_rounded_corners"/>
```

Comment:
I think what we want here is the [default](https://github.com/aosp-mirror/platform_frameworks_base/blob/master/core/res/res/drawable/btn_default_material.xml) ripple color?

Reply:
We might be able to deprecate this drawable given that we are on API 23, check my other comment 🙂

Reply:
Ah good catch